packages/shared/src/store/storage/storageActions.ts

```diff
@@ -104,22 +104,33 @@ const performUpload = async (asset: ImagePickerAsset, isWeb = false) => {
       credentials,
       forcePathStyle: true,
     });
+
+    const headers = {
+      'Content-Type': asset.mimeType ?? 'application/octet-stream',
+      'Cache-Control': 'public, max-age=3600',
```

Comment:
looks like we drop all these headers if it's not digital ocean? Do hosted uploads still work without `Content-Type` and `Cache-Control`?

Reply:
Good catch, looking into it.